extensions/data-plane/data-plane-http/build.gradle.kts

```diff
@@ -18,13 +18,16 @@ val rsApi: String by project
 
 plugins {
     `java-library`
+    `java-test-fixtures`
 }
 
 dependencies {
     api(project(":extensions:data-plane:data-plane-spi"))
     implementation(project(":common:util"))
     implementation("com.squareup.okhttp3:okhttp:${okHttpVersion}")
     implementation("net.jodah:failsafe:${jodahFailsafeVersion}")
+    testFixturesImplementation("com.github.javafaker:javafaker:1.0.2")
```

Comment:
extract version into `gradle.properties`?

Reply:
Good suggestion, I will do it now.

Reply:
Update it.